### PR TITLE
[WIP] first step towards new style imports

### PR DESCRIPTION
**Pull request by looooo**
- adds the possibility to import from __freecad__
- installs __freecad__ to a std python path -> simplifies importing freecad from python (when launched without FreeCAD-gui)

### CODE REVIEW

**Comment by wwmayer** [comment]
make install fails with: file INSTALL cannot find "/home/travis/build/FreeCAD/FreeCAD/src/Mod/Part/Part_SRCS".

File: src/Mod/Part/CMakeLists.txt [hunk 1]
```diff
@@ -4,48 +4,74 @@ if(BUILD_GUI)
...
+
 INSTALL(
     FILES
-        Init.py
-        InitGui.py
-        MakeBottle.py
-        TestPartApp.py
-        TestPartGui.py
-        JoinFeatures.py
+        Part_SRCS
```
Comment (wwmayer) [inline src/Mod/Part/CMakeLists.txt]: Should be ${Part_SRCS}

File: src/Mod/Part/CMakeLists.txt [hunk 2]
```diff
@@ -4,48 +4,74 @@ if(BUILD_GUI)
...
     FILES
-        BOPTools/__init__.py
-        BOPTools/GeneralFuseResult.py
-        BOPTools/JoinAPI.py
-        BOPTools/JoinFeatures.py
-        BOPTools/ShapeMerge.py
-        BOPTools/SplitAPI.py
-        BOPTools/SplitFeatures.py
-        BOPTools/Utils.py
+        BOPTools_SRCS
```
Comment (wwmayer) [inline src/Mod/Part/CMakeLists.txt]: Should be ${AttachmentEditor_SRCS}

File: src/Mod/Part/CMakeLists.txt [hunk 3]
```diff
@@ -4,48 +4,74 @@ if(BUILD_GUI)
...
 )
 
 INSTALL(
     FILES
-        AttachmentEditor/__init__.py
-        AttachmentEditor/Commands.py
-        AttachmentEditor/FrozenClass.py
-        AttachmentEditor/TaskAttachmentEditor.py
-        AttachmentEditor/TaskAttachmentEditor.ui
+        AttachmentEditor_SRCS
```
Comment (wwmayer) [inline src/Mod/Part/CMakeLists.txt]: Should be ${AttachmentEditor_SRCS}

File: src/Mod/Part/CMakeLists.txt [hunk 4]
```diff
@@ -4,48 +4,74 @@ if(BUILD_GUI)
...
     DESTINATION
         Mod/Part/BOPTools
        )
 
 INSTALL(
     FILES
-        CompoundTools/__init__.py
-        CompoundTools/_CommandCompoundFilter.py
-        CompoundTools/CompoundFilter.py
+        CompoundTools_SRCS
```
Comment (wwmayer) [inline src/Mod/Part/CMakeLists.txt]: Should be ${CompoundTools_SRCS}

**Comment by looooo** [comment]
thanks @wwmayer  for reviewing. should be fixed now.

**Comment by looooo** [comment]
locally I still have a problem with:
```
Traceback (most recent call last):
  File "/home/lo/conda/envs/fc_py36/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/home/lo/conda/envs/fc_py36/lib/python3.6/unittest/case.py", line 601, in run
    testMethod()
  File "/home/lo/conda/conda-bld/freecad_debug_1499070995666/work/Mod/Fem/TestFem.py", line 518, in test_static_freq_analysis
    fea.reset_all()
  File "/home/lo/conda/conda-bld/freecad_debug_1499070995666/work/Mod/Fem/FemTools.py", line 109, in reset_all
    self.reset_mesh_color()
  File "/home/lo/conda/conda-bld/freecad_debug_1499070995666/work/Mod/Fem/FemTools.py", line 91, in reset_mesh_color
    self.mesh.ViewObject.setNodeColorByScalars(node_numbers, zero_values)
Base.FreeCADError: PyArg_ParseTuple failed. Invalid arguments used with setNodeByScalars
```
But I have no idea how this is related to my changes. lets look what travis says...

**Comment by looooo** [comment]
https://github.com/FreeCAD/FreeCAD/pull/946

**Pull request closed without merging**